data-syncs/installation-and-maintenance/installing-the-worker-listener.md

```diff
@@ -1,17 +1,17 @@
-# Installing the Worker/Listener
+# Install the Worker/Listener
 
 ## Overview
 
-The Worker and Listener are important components for using Real-Time Data syncs. At a high level, the Worker/Listener follows this process:
+The Worker and Event Listener are vital components for real-time Data syncs. Here's a brief overview of how the Worker/Event Listener operates:
 
-- Once the Listener is successfully subscribed, it waits for events from streaming source.
-- The Listener receives a message from a streaming source and pushes it to SQL Server Broker.
-- The Worker then picks up message from SQL Server Broker.
-- The Worker fetches the matching record from the target based on the sync key.
-- If there are changes detected, the Worker pushes them to the target system. Successes and failures are logged in the worker's log file.
+- The Event Listener, once successfully subscribed, awaits events from the streaming source.
+- Upon receiving a message from a streaming source, the Event Listener pushes it to the SQL Server Broker.
+- The Worker then retrieves the message from the SQL Server Broker.
+- The Worker fetches the corresponding record from the target based on the sync key.
+- If changes are detected, the Worker pushes them to the target system. Both successes and failures get logged in the worker's log file.
```

Comment:
This is only valid for a full-file sync strategy.

In DELTA, the target records are not considered and the action is either always insert or provided by a separate action column in the source (which only works for REST/SOAP destinations AFAIK)